packages/x-data-grid/src/components/toolbar/GridToolbarDensitySelector.tsx

```diff
@@ -8,7 +8,7 @@ import MenuItem from '@mui/material/MenuItem';
 import ListItemIcon from '@mui/material/ListItemIcon';
 import { gridDensitySelector } from '../../hooks/features/density/densitySelector';
 import { GridDensity } from '../../models/gridDensity';
-import { isHideMenuKey, isTabKey } from '../../utils/keyboardUtils';
+import { isHideMenuKey} from '../../utils/keyboardUtils';
```

Comment:
You can run `pnpm run prettier` and `pnpm run eslint --fix` to fix linting errors (missing space before `}`).

Reply:
thanks for review @romgrk , done https://github.com/mui/mui-x/pull/14170/commits/d197a0afd7c1357d8996d8eb844e9d237af150bb